examples/step-65/step-65.cc

```diff
@@ -161,58 +164,66 @@ namespace Step65
-  // that the 6 faces on the surface of the shell without a manifold attached
-  // would degenerate to the surface of a cube. What we are still missing is
-  // the radius of the outer shell boundary. Since we desire a cube of extent
+  // vertices coincide so as to allow the two grid to be merged. The grid coming
+  // out of GridGenerator::hyper_shell fulfills the requirements on the inner
+  // side in case it is created with $2d$ coarse cells (6 coarse cells in 3D
+  // which we are going to use) -- this is the same number of cells as there are
+  // boundary faces for the ball. For the outer surface, we use the fact that
+  // the 6 faces on the surface of the shell without a manifold attached would
+  // degenerate to the surface of a cube. What we are still missing is the
+  // radius of the outer shell boundary. Since we desire a cube of extent
   // $[-1, 1]$ and the 6-cell shell puts its 8 outer vertices at the 8
   // opposing diagonals, we must translate the points $(\pm 1, \pm 1, \pm 1)$
   // into a radius: Clearly, the radius must be $\sqrt{d}$ in $d$ dimensions,
   // i.e., $\sqrt{3}$ for the three-dimensional case we want to consider.
   //
-  // Thus, we have a plan: After creating the inner triangulation for the ball
-  // and the one for the outer shell, we merge those two grids but remove all
-  // manifolds from the resulting triangulation to ensure that we have full
-  // control over manifolds. In particular, we want additional points added on
-  // the boundary during refinement to follow a flat manifold description. To
-  // start the process of adding more appropriate manifold ids, we assign the
-  // manifold id 0 to all mesh entities (cells, faces, lines), which will
-  // later be associated with the TransfiniteInterpolationManifold. Then, we
-  // must identify the faces and lines that are along the sphere of radius 0.5
-  // and mark them with a different manifold id to then assign a
-  // SphericalManifold to those. We will choose the manifold id of 1. Since we
-  // have thrown away all manifolds that pre-existed after calling
-  // GridGenerator::hyper_ball(), we manually go through the cells of the mesh
-  // and all their faces. We have found a face on the sphere if all four
-  // vertices have a radius of 0.5, or, as we write in the program, have
-  // $r^2=0.25$. Note that we call `cell->face(f)->set_all_manifold_ids(1)` to
-  // set the manifold id both on the faces and the surrounding
-  // lines. Furthermore, we want to distinguish the cells inside the ball and
-  // outside the ball by a material id for visualization according to the
-  // picture in the introduction.
+  // Thus, we have a plan: After creating the inner triangulation for
+  // the ball and the one for the outer shell, we merge those two
+  // grids but remove all manifolds that the functions in
+  // GridGenerator may have set, from the resulting triangulation, to
```

Comment:
```suggestion
  // GridGenerator may have set from the resulting triangulation, to
```

Reply:
I should add that I want to remove the comma because the sub-clause in the line above also starts without comma.